File: Objetos_2_java/src/main/java/objetos2/java/gestionBiblioteca/Libro.java
package objetos2.java.gestionBiblioteca;

import java.util.UUID;

public class Libro {
    String titulo;
    String autor;
    boolean prestado;
    UUID isbn = UUID.randomUUID();

    public Libro(String titulo, String autor) {
        this.titulo = titulo;
        this.autor = autor;
        this.prestado = false;
    }

    public void mostrarInfo(){
        System.out.println("Titulo: " + titulo + ", Autor: " + autor + ", Prestado: " + prestado + ", ISBN: " + isbn);
    }
}

File: Objetos_2_Kotlin/src/main/kotlin/gestionBiblioteca/Libro.kt
package gestionBiblioteca

import java.util.UUID

data class Libro(
    val titulo: String,
    val autor: String,
    var prestado: Boolean = false,
    val isbn: UUID = UUID.randomUUID()

){
    override fun toString(): String {
        return "Titulo: $titulo, Autor: $autor, isbn: $isbn"
    }
}
